src/site/xdoc/server/dev-build.xml

```diff
@@ -158,14 +155,41 @@
 
     </subsection>
 
+    <section name="Build James from IntelliJ IDEA">
+
+          <p>This section has been created using IntelliJ IDEA 12 and Maven 3. And downloading the master version of the server
+              from <code>http://git-wip-us.apache.org/repos/asf/james-project.git/</code>.
```

Comment:
can you provide apache github link instead?

Reply:
No I won't. Apache git should be the channel of distribution.

Github mirror is just a convenience.